src/test/java/org/cadixdev/mercury/test/RemappingTests.java

```diff
@@ -61,16 +61,18 @@ void remap() throws Exception {
         this.copy(in, "test/ObfClass.java");
         this.copy(in, "JavadocTest.java");
         // - Test 2
+        this.copy(in, "ParameterTest.java");
```

Comment:
There is a comment block above with all the tests described, please update that :)